### ISSUE TITLE
Custom splash screen appearing incorrectly on Android 

### ISSUE THREAD

**Issue by ALBATI-2005** [godotengine/godot#63456]
### Godot version

3.4.3

### System information

Android, GLES2

### Issue description

For older versions of Android, the CUSTOM splash screen simply appears for a microsecond or so then disappears. For newer versions, the CUSTOM splash screen appears alright but is not scaled properly to fill whole screen of the phone, despite checking on the box "Fullsize" under the Boot Splash in Project Settings. Also, notice emphasis on CUSTOM, since the preset splash screen works just fine.

### Steps to reproduce

Simple, just change the splash screen, even to a blank colour picture, the export the project to apk and try the game. Try a game with lots of resources to see how the splash screen malfunctions properly.

**Comment by Calinou** [comment]
Related to https://github.com/godotengine/godot/issues/62863 (possible duplicate?).

> For older versions of Android

Which Android versions did you test this on? Did you find an Android version where the problem doesn't occur?

**Comment by ALBATI-2005** [comment]
> Related to #62863 (possible duplicate?).
> 
> > For older versions of Android
> 
> Which Android versions did you test this on? Did you find an Android version where the problem doesn't occur?

I tested Android 5 as an older version, and Android 11 as a newer version. The results are as I explained above, Android 5 having the splash screen disappear, and Android 11 having a smaller splash screen. I didn't test any other Android versions, hence I don't know if it works for newer or older versions.

As for whether it's a replicate, I did see some people discussing it on Reddit, but never found an answer. Also, I believe I'm using the latest stable version of Godot, so as far as stable versions go, the problem is not fixed yet. 

Thanks for taking the time to consider this issue!
